blogs/big_data/data_visualization/blog.md

```diff
@@ -0,0 +1,252 @@
+
+https://www.tableau.com/products/desktop/download
+
+If you would like to download the dataset I used, please follow the link below and select New_York_Times >> covid-19-county-level-data.csv and convert the csv file into an excel file.
+
+https://www.kaggle.com/roche-data-science-coalition/uncover
+
+Once you open Tableau, you need to connect to a data source. 
+We will use a Microsoft Excel file for this dashboard.
+
+![](https://i.imgur.com/UUjUMPf.jpg)
+
+After connection, this is my raw data. I'm using data on COVID-19 cases by county here:
+
+![](https://i.imgur.com/5o0HVof.jpg)
+
+
+
+Step 2: Create a table with filters
+- 
+In the first sheet, we will create a COVID-19 stat table with state and county filters.
+This step is broken into three parts:
+* Create table
+* Remove abc column
+* Add filters
+
+**1. Create table**
+
+This is what you will see on the left side.
+Dimensions display Date, State, and County and Measures display numeric columns, such as Cases and Deaths.
+
+![](https://i.imgur.com/0GSeyCd.jpg)
+
+In the table, we would like to display Date, State, County, Cases, and Deaths columns. So we will drag those column names into "Rows" section.
+
+![](https://i.imgur.com/tLH334O.jpg)
+
+But the table looks weird since it thinks the values here are continuous.
+We need to set Cases, Deaths columns to "Discrete."
+
+![](https://i.imgur.com/YIseNHc.jpg)
+
+Once we change Cases and Deaths to Discrete, this is what it looks like!
+
+This is what we want, except that there is an Abc column. This Abc column will appear every time you create a table in Tableau. So, let's figure out how to remove the Abc column!
+
+![](https://i.imgur.com/d3lQDWo.jpg)
+
+
+**2. Remove Abc column**
+
+This part will cover how to remove the Abc column step by step.
+But, feel free to check out the video below if you would like to follow a video tutorial.
+
+https://www.youtube.com/watch?v=6FTZ6TnYaF0
+
+First, click any abc column and click Format.
+
+![](https://i.imgur.com/NZ6WGTx.jpg)
+
+Change ***Default*** Pane color to White.
+
+![](https://i.imgur.com/BcWDLzU.jpg)
+
+This is what it looks like after you change the color.
+
+![](https://i.imgur.com/ceout1e.jpg)
+
+Choose the column on the left of Abc and click Format.
+
+![](https://i.imgur.com/UqUSaab.jpg)
+
+
+Under the “Borders” (Square icon), set Column Divider Pane to None.
+
+![](https://i.imgur.com/ZVc527t.jpg)
+
+Lastly, shrink the width of the blank column. This is the final table!
+
+![](https://i.imgur.com/nTLe6cI.jpg)
+
+**3. Add filters**
+
+Now, let’s add State and County filter. (Choose Select All from List)
+
+![](https://i.imgur.com/OjVLHO0.jpg)
+
+Right click on the State filter and choose "Show Filter."
+
+![](https://i.imgur.com/xObIbNK.jpg)
+
+This filter shows all of the states. It's too lengthy!
+
+![](https://i.imgur.com/yMBKQLd.jpg)
+
+Right click the tiny dropdown in the State filter and change it to the form you like - I choose Single Value dropdown
```

Comment:
Could you be more specific for "tiny dropdown"? I wasn't sure where it was in the photos. Is it the drop down arrow?